src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -6243,7 +6306,11 @@ private void LoadSSPILibrary()
                     {
                         // use local for ref param to defer setting s_maxSSPILength until we know the call succeeded.
                         UInt32 maxLength = 0;
+#if MANAGED_SNI
+                        if (0 != SNIProxy.Singleton.InitializeSspiPackage(ref maxLength))
+#else
                         if (0 != SNINativeMethodWrapper.SNISecInitPackage(ref maxLength))
+#endif // MANAGED_SNI
```

Comment:
Once the managed code has feature parity with our native code, these #if checks will no longer be necessary and can be removed.